### ISSUE TITLE
holdings from other library appearing in local view

### ISSUE THREAD

**Issue by patrickzurek** [CARLI/vufind#12]
JIRA issue created by: Jessica Gibson (gibsonjc)
Originally opened: 2016-03-10 09:52 AM

In the local view "Local Catalog Only" of KCC, why do I also see UIU holdings displaying?

https://vufind2-test.carli.illinois.edu/vf-kcc/Record/(KCCdb)44730

Is it because this KCCdb BIB has an 035 |a (UIUdb)4412655 in it?

This indicates to me that KCC catalogers must have copied the bib from UIUdb when they cataloged it.  This will happen a fair amount, I think.

"All I-Share Libraries" version:
https://vufind2-test.carli.illinois.edu/all/vf-kcc/Record/205072625

**Comment by patrickzurek** [comment]
JIRA User: Chris Delis (cedelis)
JIRA Timestamp: 2016-03-10 09:54 AM

Yes

**Comment by patrickzurek** [comment]
JIRA User: Chris Delis (cedelis)
JIRA Timestamp: 2016-03-10 10:33 AM

I think I've found a good way to solve this. Currently both the shared catalog and each local catalog share the same exact XCNCIP2 config file. While it was nice to not have to customize it for each local instance, it seems we now need to. The shared catalog configures a list of "valid" agencies (all of CARLI's member libraries). VuFind2 ignores 035s that do not belong in this list. One solution is to create a list containing just the single library for the local catalog. I've manually tested this for KCC and it does filter out UIU's holdings from KCC's catalog.
